package.json

```diff
@@ -86,7 +89,7 @@
       "git add"
     ],
     "README.md": [
-      "npm run docs:toc",
+      "npm run docs",
       "git add"
     ]
```

Comment:
I think we want to update the node version and dependencies as well....

I see `.nvmrc` references to the `lts/fermium`, which is node `v14`.

and package.json references a really old engine:

```json
  "engines": {
    "node": ">=4.4.0"
  },
```

Reply:
Also, it is a good thing to replace `vjsstandard` with `eslint`.